stylelint-polaris/index.js

```diff
@@ -43,5 +50,34 @@ module.exports = {
       },
       {severity: 'warning'},
     ],
+    [mediaQueriesAllowedList]: [
+      {
+        // Allowed media types and media conditions
+        // https://www.w3.org/TR/mediaqueries-5/#media
+        allowedMedia: [
+          'print',
+          'screen',
+          '-ms-high-contrast',
+          'forced-colors',
+          // TODO: Add utility to @shopify/polaris-tokens to getMediaConditionNames
+          '$p-breakpoints-xs-up',
+          '$p-breakpoints-xs-down',
+          '$p-breakpoints-xs-only',
+          '$p-breakpoints-sm-up',
+          '$p-breakpoints-sm-down',
+          '$p-breakpoints-sm-only',
+          '$p-breakpoints-md-up',
+          '$p-breakpoints-md-down',
+          '$p-breakpoints-md-only',
+          '$p-breakpoints-lg-up',
+          '$p-breakpoints-lg-up',
```

Comment:
$p-breakpoints-lg-up and $p-breakpoints-xl-only are repeated twice while $p-breakpoints-lg-only and $p-breakpoints-xl-up are absent

Reply:
https://github.com/Shopify/polaris/pull/6438#discussion_r910546738